src/posix/platform/trel.cpp

```diff
@@ -191,7 +192,12 @@ static otError SendPacket(const uint8_t *aBuffer, uint16_t aLength, const otSock
 exit:
     otLogDebgPlat("[trel] SendPacket([%s]:%u) err:%s pkt:%s", Ip6AddrToString(&aDestSockAddr->mAddress),
                   aDestSockAddr->mPort, otThreadErrorToString(error), BufferToString(aBuffer, aLength));
-
+    ++sCounters.mTxPackets;
+    sCounters.mTxBytes += aLength;
```

Comment:
add `else` after above `if (ret != aLength)`  and move these into `else` block?

Reply:
OK, fixed, my previous assumption was that mTxPackets measures all the packets we sent or tried to sent this interface.

Reply:
`uint64_t mTxPackets; ///< Number of packets transmitted through TREL.` The comment here should indicate counting what is transmitted successfully over trel link. and there is separate mTxFailure for transmission failure